starlight/templates/profile_views/new_competency.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% load i18n %}
+
+{% block content %}
+
+<h1 class="ui black image header">
+    <div class="header">
+        New Competency
+    </div>
+</h1>
+
+<div class="ui form">
+    <form action="." method="post" class="ui large form" id="competency_form">
+        {% csrf_token %}
+        <h1 class="ui black image header">
+            <div class="content">
+                {% csrf_token%}
+                {{form}}
```

Comment:
Let's try to use Semantic UI form widgets here. We should be able to do so using manual field rendering with Django forms. Let me know if you need help with this!